.github/workflows/angular-components.yml

```diff
@@ -1,6 +1,12 @@
 name: Angular components CI
 
-on: [push, pull_request]
+on:
+  push:
+    paths:
+    - '**.ts'
+  pull_request:
+    paths:
+    - '**.ts'
```

Comment:
would also add *.css and *.html here since those are part of Angular code as well